src/rebar_prv_compile.erl

```diff
@@ -217,6 +217,9 @@ copy(OldAppDir, AppDir, Dir) ->
 
 %% TODO: use ec_file:copy/2 to do this, it preserves timestamps and
 %% may prevent recompilation of files in extra dirs
+copy(Source, Source) ->
+    %% someone specified a src dir inside _build. weird but ok
+    ok;
```

Comment:
Sounds like a thing where we should crash? I'm not sure what use case is ever valid for that, unless they'Re trying to run deps' suites or something?